lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -130,18 +126,26 @@ define((require, exports, module) ->
         @lastChangeTime = null
       else
         @currentDelayedUpdateId = null
-        if @contextPreviewExists
-          { row: cursorRow, column: cursorColumn } = @editor.getCursorPosition()
-          curContext = LatexParsingContext.getContext(@editor.getSession(), cursorRow, cursorColumn)
-          if curContext != "equation"
-            @destroyContextPreview()
-          else
-            @updateRange()
+        { row: cursorRow, column: cursorColumn } = @editor.getCursorPosition()
```

Comment:
Do you really need destructuring here? just `cursor = @editor.getCursorPosition` and subsequent `cursor.row` and `cursor.column` look not less readable (in fact,more readable, as to my taste)